core/.bazelrc

```diff
@@ -1,4 +1,28 @@
+build:cache --remote_cache=grpcs://remote.buildbuddy.io
+build:cache --experimental_remote_cache_compression
+build:cache --experimental_remote_cache_async
+build:cache --remote_timeout=3600
+
+build:bes --bes_results_url=https://app.buildbuddy.io/invocation/
+build:bes --bes_backend=grpcs://remote.buildbuddy.io
+
+build:pr --config=cache
+build:pr --noremote_upload_local_results
+build:pr --remote_download_toplevel
+build:pr --verbose_failures
+build:pr --disk_cache=
+build:pr --flaky_test_attempts=2
+
+build:ci --config=cache
+build:ci --config=bes
+build:ci --remote_upload_local_results
+build:ci --remote_download_toplevel
+build:ci --verbose_failures
+build:ci --disk_cache=
+build:ci --flaky_test_attempts=2
+
 build --host_platform=@rules_nixpkgs_core//platforms:host
+build --platforms=@rules_nixpkgs_core//platforms:host
```

Comment:
Did you add this line on purpose? This should not hurt, but all it does is add the `support_nix` constraint on top of the `@local_config_platform//:host` platform. The target platform should not need this constraint, I think this will not be checked.

Reply:
Honestly don't remember why I added it, will remove it again.